integration-tests/src/test/java/oracle/kubernetes/operator/ItJrfPvWlst.java

```diff
@@ -14,133 +14,167 @@
-  * RCU.
-  *
-  * @throws Exception - if an error occurs when load property file or create DB pod
-  */
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   */
   @BeforeAll
-  public static void staticPrepare() throws Exception {
+  public static void staticPrepare() {
     namespaceList = new StringBuffer();
     testClassName = new Object() {
     }.getClass().getEnclosingClass().getSimpleName();
-    // initialize test properties 
-    initialize(APP_PROPS_FILE, testClassName);  
+    
+    LoggerHelper.getLocal().log(Level.INFO, "setting up properties, directories for: {0}", 
+        testClassName);
+    assertDoesNotThrow(() -> initialize(APP_PROPS_FILE, testClassName),
+        "Failed: initial setup");   
   }
 
   /**
-   * Prepare test.
-   * @throws Exception on failure
+   * This method gets called before every test. It creates the result/pv root directories
+   * for the test. It also creates RCU schema, operator.
    */
   @BeforeEach
-  public void prepare() throws Exception {
-    if (QUICKTEST) {
-      createResultAndPvDirs(testClassName);
-      
-      TestUtils.exec(
-          "cp -rf " 
-          + BaseTest.getProjectRoot() 
-          + "/kubernetes/samples/scripts " 
-          + getResultDir(),
-          true);
-      //delete leftover pods caused by test being aborted
-      DbUtils.deleteRcuPod(getResultDir());
-      DbUtils.stopOracleDB(getResultDir());
-       
-      DbUtils.startOracleDB(getResultDir());
-      DbUtils.createRcuSchema(getResultDir(),rcuSchemaPrefix);
+  public void prepare() {
+
+    //createResultAndPvDirs(testClassName);
+    LoggerHelper.getLocal().log(Level.INFO, "Creating result/pv root directories");
+    assertDoesNotThrow(() -> createResultAndPvDirs(testClassName),
+        "Failed: createResultAndPvDirs");
     
-      // create operator1
-      if (operator1 == null) {
-        Map<String, Object> operatorMap = createOperatorMap(getNewSuffixCount(),
-            true, testClassName);
-        operator1 = TestUtils.createOperator(operatorMap, Operator.RestCertType.SELF_SIGNED);
-        Assertions.assertNotNull(operator1);
-        domainNS = ((ArrayList<String>) operatorMap.get("domainNamespaces")).get(0);
-        namespaceList.append((String)operatorMap.get("namespace"));
-        namespaceList.append(" ").append(domainNS);
-      }
-    }  
+    LoggerHelper.getLocal().log(Level.INFO, "Copying sample dir to the result dir");
+    assertDoesNotThrow(() -> TestUtils.exec(
+        "cp -rf " 
+         + BaseTest.getProjectRoot() 
+         + "/kubernetes/samples/scripts " 
+         + getResultDir(),
+         true),
+         "Failed: Copy sample dir to the result dir");
+    
+    //start DB and create RCU
+    dbNamespace = "db" + String.valueOf(getNewSuffixCount());
```

Comment:
looks like a heap of duplicated code here

Reply:
I removed the duplicate logging msg and also moved createNamespace() into  DbUtils.setupRCUdatabase. 
For every JRF test case we need to create RCU schema once.